Documentation/Rust Style.md

```diff
@@ -41,7 +39,11 @@ In `impl`s, always put public functions before private ones.
 
 If a crate has that feature, make sure your code does not produce any new errors when compiling with `--features=clippy`. If you don't agree with a [Clippy lint](https://github.com/Manishearth/rust-clippy#lints), discuss it with the team before explicitly adding an `#[allow(lint)]` attribute.
 
-For clippy, we currently use Rust nightly-2016-12-19 and Clippy 0.0.104.
+For clippy, we currently use Clippy 0.0.120 and nightly available on 17-03-2017 which is:
+```rust
+rustc --version
+rustc 1.17.0-nightly (0aeb9c129 2017-03-15)
+```
 
 **Note for Windows users:** Due to a recent bug in rustup, you may get a missing dll error when trying to run `cargo clippy`.  In this case, you can work around the issue by modifying your `PATH` environment variable:
 
```

Comment:
We also need to update line 51 from `nightly-2016-12-19-x86_64-pc-windows-gnu` to `nightly-2017-03-16-x86_64-pc-windows-gnu`.